jbpm-services/jbpm-kie-services/src/main/java/org/jbpm/kie/services/impl/RuntimeDataServiceImpl.java

```diff
@@ -257,19 +257,18 @@ protected void applyDeploymentFilter(Map<String, Object> params) {
     protected void applySorting(List<ProcessDefinition> input, final QueryContext queryContext) {
     	if (queryContext != null && queryContext.getOrderBy() != null && !queryContext.getOrderBy().isEmpty()) {
     		Collections.sort(input, new Comparator<ProcessDefinition>() {
-
-				@Override
-				public int compare(ProcessDefinition o1, ProcessDefinition o2) {
-					if ("ProcessName".equals(queryContext.getOrderBy())) {
-						return o1.getName().compareTo(o2.getName());
-					} else if ("ProcessVersion".equals(queryContext.getOrderBy())) {
-						return o1.getVersion().compareTo(o2.getVersion());
-					} else if ("Project".equals(queryContext.getOrderBy())) {
-						return o1.getDeploymentId().compareTo(o2.getDeploymentId());
-					}
-					return 0;
-				}
-			});
+                @Override
+                public int compare(ProcessDefinition o1, ProcessDefinition o2) {
+                    if ("ProcessName".equals(queryContext.getOrderBy())) {
+                        return o1.getName().compareToIgnoreCase(o2.getName());
```

Comment:
I think the formatting changes in here are not necessary, would suggest to simply change the affected lines to have a simplified diff.